src/pseudo_blockarray.jl

```diff
@@ -60,6 +60,32 @@ PseudoBlockArray(blocks::R, block_sizes::Vararg{AbstractVector{Int}, N}) where {
     PseudoBlockArray(blocks, Vector{Int}.(block_sizes)...)
 
 
+
+@inline function PseudoBlockArray{T}(block_sizes::BlockSizes{N}) where {T, N}
```

Comment:
Presumably all these new `PseudoBlockArray` constructors should use `uninitialized`?

Reply:
Yes I think you are right, even if the package supports 0.6.

My only hesitation is that sometimes changes in Julia master are backtracked on, but in this case that’s unlikely to happen.